### ISSUE TITLE
[Adaptive Card iOS - Table.FirstRowAsHeaders.json]: While navigating using swipe gesture Rating stars are not accessible.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#187]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 17.5.1
Application: Adaptive Cards
App Version: Version 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**

1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.5" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Input.Rating.Actions.Json' and activate it.
7. The Card will open.
8. Navigate to the rating star using swipe gesture and observe the issue.

**Actual Result:**
While navigating to the Rating Star using the swipe gesture, Rating stars are not accessible through the swipe gesture. On performing left or right swipe, voice over focus is landing on the next control.

**Observations:**
Using manually tapping on the stars, user is able to select the rating stars individually.

**Expected Result:**
While navigating using the swipe gesture, Rating star should be individually accessible through the swipe gesture.

**User Impact:**
Screen Reader user will be impacted as they will not be able to access the control while navigating using the swipe gesture due to this they will not be able to provide the rating to the user.

**WCAG Reference:**
https://www.w3.org/TR/WCAG21/#meaningful-sequence

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/7c862920-c53c-453d-9de5-7854e32f6258

### Card JSON

```JSON
NA
```

**Comment by VikrantSingh01** [comment]
Abhishek, Can you take a look at this rating a11y bug from rating component. Lets target the fix by EOW.

**Comment by wyAbhishek** [comment]
> Abhishek, Can you take a look at this rating a11y bug from rating component. Lets target the fix by EOW.

Sure, checking this.
